packages/vue/src/components/molecules/SfGallery/SfGallery.js

```diff
@@ -30,54 +30,108 @@ export default {
       type: Object,
       default() {
         return {
-          type: "slider",
+          type: 'slider',
           autoplay: false,
           rewind: false,
-          gap: 0
+          gap: 0,
         };
-      }
+      },
+    },
+    /**
+     * Image zoom inside or overlap the stage
+     */
+    outsideZoom: {
+      type: Boolean,
+      default: false,
+    },
+    /**
+     * Toogle for image zoom or overlap the stage
+     */
+    disableZoom: {
+      type: Boolean,
+      default: false
     }
   },
 
   data() {
     return {
+      positionStatic: '',
+      eventHover: '',
+      pictureSelected: '',
       glide: null,
-      activeIndex: this.current - 1
+      activeIndex: this.current - 1,
     };
   },
 
   computed: {
     mapPictures() {
       // map images to handle picture tags with SfImage
-      return this.images.map(({ normal, big }) => {
-        return {
-          small: normal,
-          normal: big
-        };
-      });
-    }
+      return this.images.map(({ normal, big }) => ({
+        small: normal,
+        normal: big,
+      }));
+    },
   },
 
   methods: {
+    positionObject(index) {
+      if (this.$refs.sfGalleryBigImage) {
+        if(this.outsideZoom){
+          return this.$refs.sfGalleryBigImage[index].$refs.imgLazy.getBoundingClientRect();
+        } else {
+          return this.$refs.sfGalleryBigImage[index].$el.getBoundingClientRect();
+        }
+      }
+      return '';
+    },
     go(index) {
       this.activeIndex = index;
       /**
        * Event for current image change (`v-model`)
        * @type {Event}
        */
-      this.$emit("click", index + 1);
+      this.$emit('click', index + 1);
       if (this.glide) {
-        this.glide.go("=" + index);
+        this.glide.go(`=${index}`);
       }
-    }
+    },
+    addImg(picture) {
+      if(!this.disableZoom){
+        const { zoom, big, normal } = picture
+        this.pictureSelected = (zoom || big || normal).url
+      }
+    },
+    changeImg($event, index) {
```

Comment:
`changeImg` do you change some image here?

Reply:
wdyt?
```
mouseover = "startZoom"
mousemove = "moveZoom"
mouseout = "removeZoom"
```
or better...
```
mouseover = "triggerMouseover"
mousemove = "triggerMousemove"
mouseout = "triggerMouseout"
```
and inside .js call another functions...

Reply:
Imo first ones sounds better (more descriptive):

```vue
mouseover="startZoom"
mousemove="moveZoom"
mouseout="removeZoom"
```

Reply:
yup, first sounds more descriptive, thank @juanfrantomaswac